app/protocols/index.js

```diff
@@ -90,6 +93,10 @@ export async function setupProtocols (session) {
   sessionProtocol.registerStreamProtocol('agregore', browserProtocolHandler)
```

Comment:
Rename the current `browserProtocolHandler` to the `agregoreProtocolHandler`, I think this will make the code more readable and more obvious.